app/connectors/CalculationDetailsConnector.scala

```diff
@@ -28,7 +28,7 @@ class CalculationDetailsConnector @Inject()(httpClient: HttpClient,
                                            (implicit ec: ExecutionContext) {
 
   def getCalculationDetails(nino: String, calculationId: String)(implicit hc: HeaderCarrier): Future[CalculationDetailResponse] = {
-    val getCalculationDetailsUrl: String = appConfig.desBaseUrl + s"/income-tax/view/calculations/liability/$nino/$calculationId"
+    val getCalculationDetailsUrl: String = appConfig.ifBaseUrl + s"/income-tax/view/calculations/liability/$nino/$calculationId"
 
     httpClient.GET(url = getCalculationDetailsUrl)(CalculationDetailsHttpReads, hc, ec)
```

Comment:
think this line should be like the other connectors like

desCall(desHeaderCarrier(getCalculationDetailsUrl))

but for IF if it needs to be IF rather than DES.